styles/landing.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+
+
+:root{
+    --primary-color:#456568;
+    --background-color:#F9F9F9;
+    
```

Comment:
you can shift this to base.css, so that you don't have to define it in every css file